### ISSUE TITLE
While playing, Back button doesn't restart the song.

### ISSUE THREAD

**Issue by WebShapedBiz** [artemanufrij/playmymusic#8]

**Comment by artemanufrij** [comment]
Hey @WebShapedBiz, it plays previos song, right?

**Comment by WebShapedBiz** [comment]
Yes it does play a previous song, it just won't go back to the start of the song.

**Comment by artemanufrij** [comment]
@WebShapedBiz could you test current master branch, please?

**Comment by WebShapedBiz** [comment]
I would love to but I know how to work with debs only.
Since I'm having my music app running all day I will test the living daylights out of this, but only with debs provided, sorry.

**Comment by artemanufrij** [comment]
No problem.
Newst behavior:
if progress > 1 second => current song will be started again (0:00). otherwise prev song will be played

**Comment by WebShapedBiz** [comment]
Thank you Artem.

**Comment by artemanufrij** [comment]
you are welcome!
